### PR TITLE
mixer is now facebook gaming

### PR DESCRIPTION
**Pull request by jbpratt**
maybe should try to improve facebook support now but that is falling low
on the totem pole

### CODE REVIEW

**Review by KenanY — APPROVED** [review]
Any changes needed in `src/util/supported-chats.js` or `src/components/StreamEmbed.jsx`?

**Comment by jbpratt** [comment]
> Any changes needed in `src/util/supported-chats.js` or `src/components/StreamEmbed.jsx`?

Ty for looking, I only grepped for `Mixer` not `mixer`

File: api/src/Channel.h [hunk 1]
```diff
@@ -26,10 +25,10 @@ constexpr folly::StringPiece kYouTubePlaylistService{"youtube-playlist"};
 constexpr folly::StringPiece kYouTubeService{"youtube"};
 
 constexpr std::array<folly::StringPiece, 12> kServices{
```
Comment (KenanY) [inline api/src/Channel.h:27]: Should we change this too?
```suggestion
constexpr std::array<folly::StringPiece, 10> kServices{
```

Comment (jbpratt) [reply api/src/Channel.h:27]: the eyes of an eagle

**Review by slugalisk — APPROVED** [review]
lgtm

File: api/tests/StreamsTest.cpp [hunk 2]
```diff
@@ -78,7 +78,6 @@ TEST(StreamsTest, TestGetAPIJSON) {
   auto streams = new Streams(db);
   auto status = Status(StatusCode::OK, "");
   auto valid = streams->Emplace(Channel::Create("test", "youtube", &status));
-  auto removed = streams->Emplace(Channel::Create("jbpratt", "mixer", &status));
```
Comment (slugalisk) [inline api/tests/StreamsTest.cpp:81]: can we switch the service? this test will fail

Comment (jbpratt) [reply api/tests/StreamsTest.cpp:81]: I deleted this

Comment (jbpratt) [reply api/tests/StreamsTest.cpp:81]: But that is incorrect, let me look again :)

**Pull request merged**